package.json

```diff
@@ -105,6 +105,8 @@
   "workspaces": {
     "packages": [
       "packages/*",
+      "test-projects/*",
```

Comment:
Can we nest this in `tests`?

Reply:
Sure, as i published this i think it might also make sense for us to colocate the tests with the test projects 
(cc: @timleslie)